Gruntfile.js

```diff
@@ -217,10 +217,6 @@ module.exports = function (grunt) {
                 files: watch.contentScope,
                 tasks: ['exec:buildContentScope', 'copy:contentScope']
             },
-            backgroundES6JS: {
-                files: watch.background,
-                tasks: ['browserify:background']
```

Comment:
Could we drop the browserify step above for the UI code too?

Reply:
I just noticed I was missing the other 'browserify:background' I don't think this had a functional change tbh, the bit I removed is just the watcher.

I've pushed to CI to see what explodes; I expect everything :D

Reply:
Simple answer is no, we should fast follow with that change though I think.